tracker/tracker-graphql/src/types.ts

```diff
@@ -0,0 +1 @@
+export type Sanitizer = (values: Record<string, any>) => Record<string, any>;
```

Comment:
does graphql data always represent a json though? Spec does not require it but its common practice. 
I think something like T => T might be better in this case

Reply:
It's a good question, I changed it with a generic type as suggested